docker/.env.example.full

```
@@ -111,6 +111,22 @@ WEB_URL="UNCONFIGURED"
 # (Optional - default: 300, Type: int)
 CAPTCHA_CODE_EXPIRE_TIME="300"
 
+# OAuth client ID issued by Google
+# (Optional - default: )
+GOOGLE_OAUTH_CLIENT_ID=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)